libraries/standard/mqtt/utest/mqtt_lightweight_utest.c

```diff
@@ -709,6 +709,8 @@ void test_MQTT_GetUnsubscribePacketSize( void )
     size_t remainingLength = 0;
     size_t packetSize = 0;
     MQTTStatus_t status = MQTTSuccess;
+    MQTTSubscribeInfo_t fourThousandSubscriptions[ 4096 ] = { 0 };
+    int i;
```

Comment:
Will using a non-bit-width specific integer type generate a warning with the `-Wextra -Wall` flags?

Reply:
There weren't any warnings when running the tests, and I don't see why that would be a warning. `int32_t` would be typedef'ed to `int` anyway in most systems, right?

This test isn't new, it's copied from test_MQTT_GetSubscribePacketSize

Reply:
I faintly remember that some compiler had complained about using `int`, but if you don't see any warning, that works.